frontend/src/shared/components/tasque-team-select/tasque-team-select.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <div class="users" *ngIf="avatars.length <= 4">
+    <a class="user-avatar" *ngFor="let item of avatars.slice(0,7); let i = index" [attr.data-index]="i">
+      <img *ngIf="item.avatarURL" class="user-avatar-img" [src]="item.avatarURL" [alt]="item.username">
+      <div *ngIf="!item.avatarURL" class="user-no-avatar" style="background-color: {{profileColors[i]}}">
```

Comment:
These ngIf conditions would always be false, as we always have value in avatarURL.

Reply:
Actually no, this works fine 
![image](https://user-images.githubusercontent.com/29006265/185049101-bf0fa066-6696-4142-9c37-add3b9ce75a3.png)